packages/babel-plugin-transform-spread/src/index.js

```diff
@@ -62,12 +62,29 @@ export default declare((api, options) => {
           return;
         }
 
-        path.replaceWith(
+        const toConsumableArray = this.addHelper("toConsumableArray");
```

Comment:
You should move this call inside the `wrapCallConsumableArray` function, otherwise we are going to reuse the same node multiple times.